deploy

```diff
@@ -20,6 +20,10 @@ COMPOSER_BIN=$(command -v composer 2>/dev/null)
 
 DOCS_REPO=https://github.com/phalcon/docs
 
+if [ -f .env ]; then
+    source .env
```

Comment:
If the file is present and DOCS_REPO is defined in `.env` then the variable will be re-defined